src/App.tsx

```diff
@@ -1,27 +1,72 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { GoodsList } from './GoodsList';
+import { Good } from './types/Good';
```

Comment:
Consider using more descriptive names for types. For example, `Good` could be renamed to `Product` or `Item`. 📝